float64 x
float64 y
float64 z
float64 rx
float64 ry
float64 rz
int32 tool
string[] paramValue

---

int32 res